### PR TITLE
Convert sp map objects to sf objects

### PR DESCRIPTION
**Pull request by rudeboybert**

### CODE REVIEW

**Comment by arrismo** [comment]
@rudeboybert for some reason on R studio it used my computer name as the commit author. I am not sure if it will effect anything technically. In other news I have finished converting SP to SF.

File: R/add.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#' Add together two numbers
```
Comment (rudeboybert) [inline R/add.R:1]: Be sure to delete any superfluous non-essential file and/or code like this. Keep things minimalist and clean

File: man/add.Rd [hunk 2]
```diff
@@ -0,0 +1,23 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (rudeboybert) [inline man/add.Rd:1]: Delete this too

File: pennlc_map.Rmd [hunk 3]
```diff
@@ -0,0 +1,121 @@
+---
```
Comment (rudeboybert) [inline pennlc_map.Rmd:1]: If this isn't needed, delete this too

File: data-raw/add_sf_data.R [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+# Adding pennLC Pennsylvania lung cancer data as shapefile
+pennLC_sf <- read_csv("data-raw/pennLC_sf.csv")
+#usethis::use_data(pennLC_sf,overwrite = TRUE)
+
+
+pennLC_sf <- st_as_sf(pennLC$spatial.polygon)
+
+# add all of the cases together by county
+pennLC_df <- pennLC$data %>%
```
Comment (rudeboybert) [inline data-raw/add_sf_data.R:17]: noteworthy point here: by summarizing by county, we're losing the race/gender/age breakdowns and only left with aggregate numbers. For each county, i.e. each polygon, we want to provide 16 values: case counts, AND population for all 2 x 2 x 4 possible race/gender/age strata

File: data-raw/add_sf_data.R [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                    york)
+
+# add county to sf object manually
+pennLC_sf$county <- county
+
+# now that we have a common value we can join
+pennLC_sf <- left_join(pennLC_sf, pennLC_df)
+
+# reorder columns so that geometry is first
+pennLC_sf <- pennLC_sf[c(3,2,1)]
```
Comment (rudeboybert) [inline data-raw/add_sf_data.R:96]: best coding practice: by using `3,2,1` you're assuming that the order of the variables is the way you want it. it's safer to use explicit names and do something like `pennLC_sf[geometry, cases, ...]`

File: R/sf_objects.R [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+#' ggplot() + 
+#' geom_sf(data = pennLC_sf)
+#' 
+#' Interactive Map using Leaflet
+#' 
+#' leaflet(pennLC_sf) %>%
+#' addPolygons()
+#' 
+#' @return \code{pennLC_sf}
+NULL
```
Comment (rudeboybert) [inline R/sf_objects.R:19]: This is a little different way to roxygen document a data set than I'm used to. Where did you see this? I'm more use to [this](https://github.com/rudeboybert/fivethirtyeight/blob/master/R/data.R#L91) style, which is below. 
```r
#' Pennsylvania Lung Cancer SF Object
#' 
#' pennLC_sf is a spatial object that can be used to create static or interactive maps.
#' 
#' @format A data frame with 56 rows representing airlines and 9 variables:
#' \describe{
#'   \item{geometry}{airline}
#'   \item{cases}{indicates that regional subsidiaries are included}
#'   \item{county}{available seat kilometers flown every week}
#' }
#' @source Aviation Safety Network \url{https://aviation-safety.net}.
"pennLC_sf"
```
Note we're prob goign to need to come back to this file after we fix the issue with the data frame being aggregated

Comment (arrismo) [reply R/sf_objects.R]: @rudeboybert  I have updated the doc file for pennLC_sf.

**Pull request merged**